models/notes.py

```diff
@@ -0,0 +1,57 @@
+from db import db
+from datetime import datetime
+from models.user import UserModel
+
+class NotesModel(db.Model):
+    __tablenname__ = "Notes"
```

Comment:
Looks like a typo in "tablename" here

Reply:
fixed it -- strange that it would work with the typo. Don't know what to make of that.